## Schedule

#### Thursday, 14 September

<table border=0 cellpadding=9 width='90%'>
<tr>
<td valign='top'>9.55</td>
<td valign='top'>Opening<br>
    <i>Damjana Kastelic<br>
    Scientific Training, CRG, Spain</i></td>
</tr>

<tr>
<td valign='top'>10.00</td>
<td valign='top'>Welcome & introduction<br>
    <i>Cedric Notredame<br>
    Comparative Bioinformatics, CRG, Spain</i></td>
</tr>

<tr>
<td valign='top'>10.10</td>
<td valign='top'>Lessons learned and new challenges<br>
    <i>Paolo Di Tommaso<br>
    Comparative Bioinformatics, CRG, Spain</i></td>
</tr>

<tr>
<td valign='top'>10.30</td>
<td valign='top'><a href='abstracts.md#standardising-swedish-genomics-analyses-using-nextflow'>Standardising Swedish genomics analyses using Nextflow</a> [<a href='https://www.slideshare.net/tallphil/standardising-swedish-genomics-analyses-using-nextflow/1'>slides</a>]<br>
    <i>Phil Ewels<br>
    National Genomics Infrastructure, SciLifeLab, Sweden</i>
    </td>
</tr>

<tr>
<td valign='top'>11.00</td>
 <td valign='top'><a href='abstracts.md#building-pipelines-to-support-african-bioinformatics'>Building Pipelines to Support African Bioinformatics</a><br>
    <i>Scott Hazelhurst<br>
    University of the Witwatersrand, Johannesburg, South Africa</i>
    </td>
</tr>

<tr>
<td valign='top'>11.30</td>
<td valign='top'>coffee break</i>
    </td>
</tr>

<tr>
<td valign='top'>12.00</td>
<td valign='top'><a href='abstracts.md#inside-out-reproducible-analysis-of-external-data-inside-containers-with-nextflow'>Inside-Out: reproducible analysis of external data, inside containers with Nextflow</a><br>
    <i>Evan Floden<br>
    Comparative Bioinformatics, CRG, Spain</i></td>
</tr>

<tr>
<td valign='top'>12.30</td>
 <td valign='top'><a href='abstracts.md#nextflow-for-chemistry---crossing-the-divide'>Nextflow for chemistry - crossing the divide</a><br>
    <i>Tim Dudgeon<br>
    Informatics Matters Ltd, UK</i>
    </td>
</tr>

<tr>
<td valign='top'>12.50</td>
 <td valign='top'><a href='abstracts.md#from-zero-to-nextflow--crgs-biocore'>From zero to Nextflow @ CRG's Biocore</a><br>
    <i>Luca Cozzuto<br>
    Bioinformatics Core Facility, CRG, Spain</i>
    </td>
</tr>

<tr>
<td valign='top'>13.10</td>
<td valign='top'><a href='https://github.com/nextflow-io/hack17/blob/master/abstracts.md#simplifying-shotgun-metagenomic-analysis-with-nextflow'>Simplifying shotgun metagenomic analysis with Nextflow</a><br>
    <i>Alessia Visconti<br>
    Department of Twins Research, King's College, UK</i></td>
</tr>

<tr>
<td valign='top'>13.30</td>
<td valign='top'>Lunch at <a href='https://goo.gl/maps/27FHPyPtUGB2'>Shoko</a> restaurant.
</tr>

<tr>
<td valign='top'>14.30<br>18.30</td>
<td valign='top'>Hackathon (CRG Bioinformatics room, 4th floor, 486) & course (Charles Darwin room)</td>
</tr>

<tr>
<td valign='top'>20.00</td>
<td valign='top'>Social dinner at <a href='https://goo.gl/maps/JJXAbxHQYmr'>La Fooditeca</a> restaurant.
 </td>
</tr>

</table>

#### Friday, 15 September

<table border=0 cellpadding=9 width='90%'>
<tr>
<td valign='top'>9.30</td>
<td valign='top'><a href='abstracts.md#computational-workflows-for-omics-analyses-at-the-international-agency-for-research-on-cancer'>Computational workflows for omics analyses at the IARC</a><br>
    <i>Matthieu Foll<br>
    International Agency for Research on Cancer (IARC), France</i></td>
</tr>

<tr>
<td valign='top'>10.00</td>
 <td valign='top'><a href='abstracts.md#medical-genetics-at-oslo-university-hospital'>Medical Genetics at Oslo University Hospital</a><br>
    <i>Hugues Fontanelle<br>
    Oslo University Hospital, Norway</i></td>
</tr>

<tr>
<td valign='top'>10.30</td>
<td valign='top'><a href='abstracts.md#using-nextflow-for-large-scale-benchmarking-of-phylogenetic-methods-and-tools'>Using Nextflow for Large Scale Benchmarking of Phylogenetic methods and tools</a><br>
    <i>Frédéric Lemoine<br>
    Evolutionary Bioinformatics, Institut Pasteur, France</i>
    </td>
</tr>

<tr>
<td valign='top'>11.00</td>
<td valign='top'>coffee break</i></td>
</tr>

<tr>
<td valign='top'>11.30</td>
<td valign='top'><a href='abstracts.md#flexible-bioinformatics-workflows-for-ngs-applications'>Flexible bioinformatics workflows for NGS applications</a><br>
    <i>Johnny Wu<br>
    Roche Sequencing, Pleasanton, USA</i></td>
</tr>

<tr>
<td valign='top'>12.00</td>
<td valign='top'><a href='abstracts.md#standardizing-life-sciences-datasets-to-improve-studies-reproducibility-in-the-european-open-science-cloud'>Standardizing life sciences datasets to improve studies reproducibility in the EOSC</a><br>
    <i>Jordi Rambla<br>
    European Genome-Phenome Archive, CRG</i></td>
</tr>

<tr>
<td valign='top'>12.20</td>
 <td valign='top'><a href='abstracts.md#putting-the-computation-in-computational-biology'>Putting the Computation in Computational Biology</a><br>
    <i>Mike Smoot<br>
    Synthetic Genomics, La Jolla, CA, USA</i></td>
</tr>

<tr>
<td valign='top'>12.40</td>
 <td valign='top'><a href='abstracts.md#unbounded-by-economics'>Unbounded By Economics</a><br>
    <i>Angel Pizarro<br>
    AWS Research and Techical Computing, Amazon, USA</i></td>
</tr>

<tr>
<td valign='top'>13.00</td>
<td valign='top'>Lunch, CRG ground floor
 </td>
</tr>

<tr>
<td valign='top'>14.00<br>18.00</td>
<td valign='top'>Hackathon (Charles Darwin room)</i></td>
</tr>

</table>
